src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -2576,6 +2576,22 @@ void ApplicationManagerImpl::CreateApplications(SmartArray& obj_array,
   }
 }
 
+void ApplicationManagerImpl::RemovePostponedDDState(
```

Comment:
I wasn't suggesting to move this to a separate function (especially not a public one), this code is small enough in scope that I think it's body could just be included in `SendDriverDistractionState`.

If you really want to make this a separate function, it should probably be genericized with a `function_id` parameter.

Reply:
@jacobkeeler Thanks. I believe, that separate function with  `function_id` as input parameter will be redundant, because this code uses only for `OnDriverDistractionID` function.
Fixed in 6ca52b7d74b3401566b039aa4ab37b153e2db767.